Dockerfile

```diff
@@ -229,33 +229,16 @@ RUN adduser www-data root
 
 # From c2cwsgiutils
 
-CMD ["c2cwsgiutils-run"]
 
 ENV TERM=linux \
   LANG=C.UTF-8 \
-  LOG_TYPE=console \
-  LOG_HOST=localhost \
-  LOG_PORT=514 \
-  SQL_LOG_LEVEL=WARN \
```

Comment:
We do not keep default values for those variables ?
They might be needed for development.ini.

Reply:
Some are moved to the project Docker file, Some other was no more used, Some are empty, after that, we steel have only C2C_SECRET and C2C_REDIS_DB which one I think that correct to don't have a default value, do you aggree?